packages/api/src/routes/medical/tcm-encounter.ts

```diff
@@ -0,0 +1,85 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import httpStatus from "http-status";
+import { listTcmEncounters } from "../../command/medical/tcm-encounter/list-tcm-encounters";
+import { updateTcmEncounter } from "../../command/medical/tcm-encounter/update-tcm-encounter";
+import { requestLogger } from "../helpers/request-logger";
+import { paginated } from "../pagination";
+import { asyncHandler, getCxIdOrFail, getFromParamsOrFail } from "../util";
+import { tcmEncounterListQuerySchema, tcmEncounterUpdateSchema } from "./schemas/tcm-encounter";
+import { NotFoundError } from "@metriport/shared";
+
+const router = Router();
+router.put("/:id", requestLogger, asyncHandler(handleUpdateTcmEncounter));
+router.get("/", requestLogger, asyncHandler(handleListTcmEncounters));
```

Comment:
ditto single route definition block, unless needed to reuse it like a deprecated route like `POST /patient/:id/consolidated`